src/components/Lookup/index.js

```diff
@@ -69,6 +71,13 @@ class Lookup extends Component {
         };
     }
 
+    componentDidMount() {
+        if (window) {
+            window.addEventListener('click', this.handleWindowClick, false);
+            window.addEventListener('touchstart', this.handleWindowClick, false);
+        }
+    }
```

Comment:
we should not use window listeners inside compoents
also this will be not needed because we must use an input instead of the chip